src/vs/platform/actions/common/actions.ts

```diff
@@ -121,72 +121,67 @@ export interface IMenuService {
 	createMenu(id: MenuId, scopedKeybindingService: IContextKeyService): IMenu;
 }
 
+export type ICommandsMap = Map<string, ICommandAction>;
+
 export interface IMenuRegistry {
 	addCommand(userCommand: ICommandAction): IDisposable;
-	getCommand(id: string): ICommandAction;
+	getCommand(id: string): ICommandAction | undefined;
 	getCommands(): ICommandsMap;
 	appendMenuItem(menu: MenuId, item: IMenuItem | ISubmenuItem): IDisposable;
 	getMenuItems(loc: MenuId): Array<IMenuItem | ISubmenuItem>;
 	readonly onDidChangeMenu: Event<MenuId>;
 }
 
-export interface ICommandsMap {
-	[id: string]: ICommandAction;
-}
-
 export const MenuRegistry: IMenuRegistry = new class implements IMenuRegistry {
 
-	private readonly _commands: { [id: string]: ICommandAction } = Object.create(null);
-	private readonly _menuItems: { [loc: number]: Array<IMenuItem | ISubmenuItem> } = Object.create(null);
+	private readonly _commands = new Map<string, ICommandAction>();
```

Comment:
I don't believe TS supports Maps natively if the target is ES5. You may need to do a workaround here unless you're outputting ES6